docs/zh/docs/insight/best-practice/integration_deepflow.md

```diff
@@ -0,0 +1,66 @@
+# 集成 DeepFlow
+
+DeepFlow 是一款基于 eBPF 的可观测性产品。它的社区版已经被集成进 Insight 中，以下是集成方式。
+
+## 前提条件
+
+- Global 集群已经安装 Insight
+- Insight 最低版本要求为 v0.23.0
+
+## 安装 DeepFlow Server
+
+1. 安装 DeepFlow chart
+
+DeepFlow Server 需要安装在 Global 集群中(它会默认安装 DeepFlow Agent)。进入 kpanda-global-cluster 集群，在左侧导航栏内点击
```

Comment:
```suggestion
    DeepFlow Server 需要安装在 Global 集群中(它会默认安装 DeepFlow Agent)。进入 kpanda-global-cluster 集群，在左侧导航栏内点击
```
Need to indent 4 space from line 14 - 30

Reply:
done